idioms/coercion-arguments.md

```diff
@@ -1,18 +1,18 @@
-# Using &str instead of &String
+# Using coercion for arguments
```

Comment:
I think it's better to emphasize the idea of using *borrowed types*, rather than the deref operations / coercions that target them.

It was always more idiomatic to use the borrowed types, even before we had deref coercions as a language feature (i.e. rust-lang/rfcs#241). The coercions just made them easier to use.